src/main/java/net/sf/jabref/logic/importer/fileformat/BibtexParser.java

```diff
@@ -74,16 +73,16 @@
     private Map<String, EntryType> entryTypes;
     private boolean eof;
     private int line = 1;
-    private final FieldContentParser fieldContentParser = new FieldContentParser(
-            FieldContentParserPreferences.fromPreferences(Globals.prefs));
+    private final FieldContentParser fieldContentParser;
     private ParserResult parserResult;
     private static final Integer LOOKAHEAD = 64;
     private final Deque<Character> pureTextFromFile = new LinkedList<>();
+    private final ImportFormatPreferences importFormatPreferences;
 
-
-    public BibtexParser(Reader in) {
+    public BibtexParser(Reader in, ImportFormatPreferences importFormatPreferences) {
```

Comment:
This is a granularity issue. As the BibtexParser is an import format it got that now.

With `FieldContentParserPreferences` it is the opposite. Yes, it can be accessed directly from `ImportFormatPreferences`, but, for historical reasons, a finer granularity was chosen there. This change would then have to be done in `LatexFieldFormatterPreferences` as well (which is not required here), so somehow it makes sense to have a separate `FieldContentParserPreferences` since it is accessed in two completely different ways.